### PR TITLE
[Bigtable] GcpCallInvoker related snippets

### PR DESCRIPTION
**Pull request by evildour**

### CODE REVIEW

File: apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2.Snippets/BigtableClientSnippets.cs [hunk 1]
```diff
@@ -104,6 +105,43 @@ public async Task Overview()
...
+        [Fact]
+        public async Task CustomCallInvoker()
+        {
+            ChannelCredentials credentials = ChannelCredentials.Insecure;
+
+            // Sample: CustomCallInvoker
+            // Create a client from a custom credentials and a GcpCallInvoker that has
+            // non-default gRPC stream/channel options.
+            // Note: the GcpCallInvoker should be long-lived while the BigtableClient
+            // instances can be emphemeral and each use the same call invoker
```
Comment (jskeet) [inline apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2.Snippets/BigtableClientSnippets.cs]: Nit: typo for ephemeral. Possibly add a period too.

File: apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2.Snippets/BigtableClientSnippets.cs [hunk 2]
```diff
@@ -104,6 +105,43 @@ public async Task Overview()
...
+            // ...
+            BigtableClient client2 = BigtableClient.Create(callInvoker, settings);
+            // ...
+
+            await callInvoker.ShutdownAsync();
+
+            // Alternatively, to just create a GcpCallInvoker will standard settings, but using non-default
+            // credentials, use null or BigtableServiceApiSettings.GetDefault() for the settings
+            settings = null;
+            callInvoker = new GcpCallInvoker(endpoint, credentials, settings.CreateChannelOptions());
```
Comment (jskeet) [inline apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2.Snippets/BigtableClientSnippets.cs]: But importantly, you can't pass in null to "new GcpCallInvoker" here, right? (This definitely *looks* like an odd call to make, until you remember it's an extension method.)

File: apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2.Snippets/BigtableClientSnippets.cs (hunk 1 above)
Comment (evildour) [reply apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2.Snippets/BigtableClientSnippets.cs]: Fixed

File: apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2.Snippets/BigtableClientSnippets.cs (hunk 2 above)
Comment (evildour) [reply apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2.Snippets/BigtableClientSnippets.cs]: Correct. Yes, it does look odd, but I wanted to show how it was ok. I'll add a note to specifically call it out so it's clear.

**Pull request merged**